packages/common/src/AppointmentUtils.ts

```diff
@@ -1,5 +1,32 @@
-import { MANUAL_DONOR_ID } from "./types";
+import { MANUAL_DONOR_ID, BookedAppointment } from "./types";
+import { DateUtils } from "./index";
 
 export function isManualDonor(donorId: string) {
   return donorId === MANUAL_DONOR_ID;
 }
+
+export function isAppointmentClosed({
+  ...bookedAppointment
+}: BookedAppointment) {
+  const DAYS_LIMIT = 2;
```

Comment:
Maybe this should be outside the function?
And please add a clarifying comment about this variable.

Reply:
Since it is a Utils folder and there is more functions in it, keeping it inside seems better

Reply:
Please see implementation in 
HospitalUtils.ts #14
DateUtils.ts #15